src/custom/state/price/actions.ts

```diff
@@ -25,12 +19,7 @@ export type QuoteError =
 
 export type SetQuoteErrorParams = UpdateQuoteParams & { error: QuoteError }
 
-export const setNewQuoteLoading = createAction<SetLoadingQuoteParams>('price/setNewQuoteLoading')
-export const setRefreshQuoteLoading = createAction<Pick<SetLoadingQuoteParams, 'loading'>>(
-  'price/setRefreshQuoteLoading'
-)
+export const getNewQuoteStart = createAction<GetQuoteParams>('price/getNewQuoteStart')
```

Comment:
why `Start` what does that mean?

Reply:
I can remove the "start". It just signals that is an async operation. It start to get the quote. Then, the when it finishes, it will either update the quote, or set an error.

I will remove the start word, i see u don't like it